### DISCUSSION TITLE
I can't find the command which acts as same as "purge" in discord.py

### DISCUSSION THREAD

**Discussion by g-cyan** [serenity-rs/serenity#1494] [Q&A] [answered]
In discord.py , the below command which I wrote works as deleting all messages which have been sent in the channel . 

    @bot.command()
    async def delete_all_messsages(ctx):
        await ctx.channel.purge(limit=None)

I want to do the same thing using serenity-rs however I've been failing to find the command which acts as same as "purge" in discord.py . 

Is it has not been implemented yet?

**Comment by Erk-** [comment, marked as answer]
Purge is not a command of the api, but one only implemented in d.py, that said it could be a good feature so this could be a feature request instead


cf. https://github.com/Rapptz/discord.py/blob/45d498c1b76deaf3b394d17ccf56112fa691d160/discord/channel.py#L412
